core/tests/protractor_utils/action.js

```diff
@@ -55,7 +55,7 @@ var select = async function(selectorName, selectorElement, optionToSelect) {
   await click(selectorName, selectorElement);
   var optionElement = selectorElement.element(
     by.cssContainingText('option', optionToSelect));
-  await optionElement.click();
+  await click(optionToSelect, optionElement);
```

Comment:
Could you make the name of the element we're clicking (first argument to `click`) more descriptive? For example: `{selectorElement}d dropdown option {optionToSelect}`?

Reply:
Done.